src/Components/Server/src/Circuits/DefaultCircuitFactory.cs

```diff
@@ -5,31 +5,33 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Security.Claims;
-using System.Text.Encodings.Web;
 using Microsoft.AspNetCore.Components.Routing;
 using Microsoft.AspNetCore.Components.Web.Rendering;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Microsoft.JSInterop;
 
 namespace Microsoft.AspNetCore.Components.Server.Circuits
 {
-    internal class DefaultCircuitFactory : CircuitFactory
+    internal class DefaultCircuitFactory
     {
         private readonly IServiceScopeFactory _scopeFactory;
         private readonly ILoggerFactory _loggerFactory;
         private readonly CircuitIdFactory _circuitIdFactory;
         private readonly CircuitOptions _options;
         private readonly ILogger _logger;
 
+        public ComponentDescriptorSerializer DescriptorSerializer { get; }
```

Comment:
This is bizzare. Is this a service? What do we get out of makign this a public property of part of the implementation?

Reply:
This was just me `ctrl+.` my way to victory. I'll make it a readonly field

Reply:
I removed it from here.

Reply:
done